clear.lua

```diff
@@ -9,10 +9,8 @@ function HandleClearCommand(Split, Player)
     if Player:HasPermission("core.admin.clear") then
         local InventoryCleared = false;
         local ClearInventory = function(OtherPlayer)
-            if (OtherPlayer:GetName() == Split[2]) then
-                OtherPlayer:GetInventory():Clear()
-                InventoryCleared = true
-            end
+            OtherPlayer:GetInventory():Clear()
+            InventoryCleared = true
```

Comment:
Perhaps show a confirmation message for critical commands like banning or clearing inventory?
"are you sure? Press Y to confirm and delete the Inventory of <Full player name here>"